src/System.Net.Primitives/src/Resources/System.Net.Primitives.rd.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library Name="*System.Net.Primitives*">
+    <Assembly Name="System.Net.Primitives">
+       <!-- VSO 449560 The following types are used by HttpListener. -->
+       <Type Name="System.Net.Cookie" Browse="Required Public" Dynamic="Required Public">
+         <!-- Used by CookieParser -->
+         <Method Name="InternalSetName" Dynamic="Required" />
```

Comment:
The Dynamic="Required Public" line above should mean you don't need extra directives for the members.